Resources/doc/cache-resolver/format_extension.rst

```diff
@@ -0,0 +1,64 @@
+
+.. _cache-resolver-format-extension:
+
+FormatExtensionResolver
+=======================
+
+The ``FormatExtensionResolver`` cannot be used by itself. Instead, it is a "decorator" for
+another resolver. It add the ability to set the correct file extension when filter converts image format.
+
+Configuration
+-------------
+
+To use this cache resolver, you must first define the cache resolver it will decorate.
+In this example, we will use the :ref:`Web Path Resolver <cache-resolver-web-path>`.
+
+Next, we need to define a service for this cache resolver and inject the web path cache resolver service to decorate.
+
+.. code-block:: yaml
+
+    # app/config/services.yml
+
+    services:
+        acme.imagine.cache.format_extension:
+            class: Liip\ImagineBundle\Imagine\Cache\Resolver\FormatExtensionResolver
+            arguments:
+                - "@acme.imagine.cache.resolver.web_path"
```

Comment:
don't we need to configure the $filterConfig as well? or does that happen automatically because of the liip_imagine.cache.resolver tag?

Reply:
You're right, I forgot it because I use autowiring in my project. Fixed.